firewall/app/main/controllers/ipController.py

```diff
@@ -0,0 +1,54 @@
+from flask import (Blueprint, request, abort)
+from flask_api import status
+from pynft import Executor
+from app.main.model.ip import IPBan
+from app.main import db
+
+from re import search
+
+bp = Blueprint('ip', __name__, url_prefix='/ip')
+PyNFT = Executor()
+
+IP_FORMAT = "^((25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])\.){3}(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])$"
+
+def validateForm(address): 
+    if (not address or address == ''):
+            return 'Address is missing', status.HTTP_400_BAD_REQUEST
+    if (not search(IP_FORMAT, address)):
+        return 'Invalid IP address', status.HTTP_400_BAD_REQUEST
+    return True
```

Comment:
should throw instead of `return 'Address is missing', status.HTTP_400_BAD_REQUEST`